src/components/Post/Post.tsx

```diff
@@ -74,15 +77,15 @@ getUserDetailRequest(apiUrl)
           {currentPost.description}
         </div>
 
-        <RowComponent>
-          {/* {currentPost.LikedNum === 0 ?
+        {/* <RowComponent>
```

Comment:
Cleanup code please. remove code that are commented out and unused.

Reply:
not sure is it useful for further enhancement

Reply:
cleaned